### DISCUSSION TITLE
Why are you releasing broken stuff?

### DISCUSSION THREAD

**Discussion by djmaze** [nocodb/nocodb#3695] [General]
Been trying to use nocodb for a few months now. I immediately realized that many things are buggy and not very usable. What's more, it seems every release breaks some feature or even removes feature that were available before.

The 0.96.x stuff seems to be hit a new record mark with so many things broken that I do not even know where to start: The table is reloaded after every edit of a field, the filtering UI is broken, it is not possible to share with non-logged-in users anymore.

Fair enough, I saw that the new stuff is a complete rewrite. But why is this stuff released even if it seems to still be WIP or not tested in any meaningful manner? There is no indication about this and I think this is not a good software development practice.

**Comment by o1lab** [comment]
hey @djmaze, I am sorry to hear you are facing an issue. Could you help us with the your current version, database type & browser you are using. We've not come across reloading issues. Filtering - please raise an issue with details. We will look into it.

And I am able to see shared view as non-logged-in user like below. And are you behind a reverse proxy ?

<img width="955" alt="Screenshot 2022-09-17 at 22 31 25" src="https://user-images.githubusercontent.com/5435402/190877032-152da056-be25-4ac1-87de-f1348998ca3f.png">

**Reply by djmaze** [reply]
Okay, I guess I need to withdraw a few of my points.

Somehow the reloading is not happening anymore now. Strange. But it seems to be connected to the fact that I have a filter activated while entering new rows. Also nocodb tries to save a row even if I did not enter all required fields yet, leading to an error message.

The shared view thing is only half working. I can still share a table with non-logged-in users, but they are not able to _edit_ any rows. That was possible before 0.96 and we were making use of that. (Because we want to keep the barrier to entry for that table as low as possible.)

The version is 0.96.4 (docker) with postgres, am I am using Firefox.

**Reply by djmaze** [reply]
Mhh.. Now I found the way to correctly share the view. I tripped again on the complexity that there are three ways to share: The `Share View` button, the `Shared View List` and the `Invite Team` button. So I can do a writable share (with the `editor` role) of the entire project but not for a single view / table. That's perplexing every time I need to figure out how to do it.

**Reply by o1lab** [reply]
> That was possible before 0.96 and we were making use of that

As you've corrected in follow up comment - this was not possible earlier either. We've a feature request for it and we will look into it. 

`Share view` - is for sharing the view
`Shared view list` - shows the list of all shared view for that particular table
`Shared base` - is for sharing entire base 

For saving new row with required field - have [raised an issue](https://github.com/nocodb/nocodb/issues/3696) to investigate. If you can add details - it will be really helpful.
